learn/get-started/write-a-restful-api-with-ballerina.md

```diff
@@ -321,6 +240,89 @@ public type InvalidIsoCodeError record {|
 In this code:
 - As in the previous example, this resource also includes its own return types. However, the basic principle behind them is as the previous example. 
 
+## The complete code
```

Comment:
This is not  available in left  nav

Reply:
@anupama-pathirage - I think it would be better to have the 3 topics below under `Understand the implementation`. These topics provide the flow to explain the full usecase in a sequential manner.

**Understand the implementation**
1. The first endpoint
2. The second endpoint
3. The complete code

Therefore, without having the complete code at the end shall we give it as above to give the full picture to the user at the beginning? WDYT?

Reply:
IMO, the current order is fine. Since this is a step-by-step guide, I  feel the complete code should be just before running. Then users can follow the steps without getting distracted by the full code which is at the beginning.  And they can verify the code which they wrote by following the steps just before running. @shafreenAnfar thoughts?

Reply:
Yeah, I also feel that the current oder is fine given that this is a step-by-step guide.